src/test/java/org/cloudfoundry/promregator/auth/AuthenticatorControllerTest.java

```diff
@@ -25,33 +27,33 @@
 	
 	@Test
 	public void testDefaultConfigurationCheck() {
-		Assert.assertNotNull(subject);
+		assertThat(subject).isNotNull();
 		
 		AuthenticationEnricher auth0 = subject.getAuthenticationEnricherById("unittestAuth0");
-		Assert.assertTrue(auth0 instanceof BasicAuthenticationEnricher);
+		assertThat(auth0).isInstanceOf(BasicAuthenticationEnricher.class);
 		
 		AuthenticationEnricher auth1 = subject.getAuthenticationEnricherById("unittestAuth1");
-		Assert.assertTrue(auth1 instanceof OAuth2XSUAAEnricher);
+		assertThat(auth1).isInstanceOf(OAuth2XSUAAEnricher.class);
```

Comment:
Improved the assertions so if the boolean check fails, the logs show the state of the variables instead of just a 'expected true was false' message